packages/astro/src/core/encryption.ts

```diff
@@ -123,3 +123,17 @@ export async function generateCspDigest(data: string, algorithm: CspAlgorithm):
 	const hash = encodeBase64(new Uint8Array(hashBuffer));
 	return `${ALGORITHMS[algorithm]}${hash}`;
 }
+
+/**
+ * Generate SHA-256 hash of buffer content for deduplication
```

Comment:
```suggestion
 * Generate SHA-256 hash of a buffer. 
```

The function doesn't have anything to do with "content duplication".

Reply:
Thanks!

Reply:
did this 
https://github.com/withastro/astro/pull/14031#discussion_r2185385298